### PR TITLE
readme: update meeting calendar

### PR DESCRIPTION
**Pull request by rootfs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,4 @@ Interested in contributing to Kepler? Follow the [Contributing Guide](CONTRIBUTI
 A full list of talks and demos about Kepler can be found [here](https://github.com/sustainable-computing-io/kepler-doc/tree/main/demos).
 
 ## Community Meetings
-Please join the biweekly community meeting, starting on 8-9AM US ET (UTC−05:00) every other Tuesdays. 
-
-[Download](https://us06web.zoom.us/meeting/tZApfumrrzgpHdwUQbkSOj0zpoGveKMMvxaD/ics?icsToken=98tyKuGsqjIqGdWWsx6PRpwIBor4c-nzmHZegqdKqS7nCBN8aFPsJu1lJLdUPfXG) the biweekly community meeting iCalendar file.
+Please join the biweekly community meetings. The meeting calendar and agenda can be found [here](https://github.com/sustainable-computing-io/community/blob/main/community-event.md)
```
Comment (SamYuan1990) [inline README.md:40]: will we update CNCF zoom in this PR?

Comment (rootfs) [reply README.md:40]: will update once we have a CNCF zoom handle

**Comment by SamYuan1990** [comment]
@rootfs , would you like me merge it? or waiting for CNCF zoom's info?

**Comment by jichenjc** [comment]
let's merge it then we can update when we have that info ..

**Pull request merged**